equinox-shipment/Domain/Shipment.fs

```diff
@@ -0,0 +1,67 @@
+﻿module Shipment
+
+open System
+
+type Shipment = { id: string; containerId: string }
+
+module Events =
+
+    let [<Literal>] CategoryId = "Shipment"
+
+    let (|ForClientId|) (clientId: string) = FsCodec.StreamName.create CategoryId clientId
+
+    type Event =
+        | ShipmentCreated    of shipmentId  : string
+        | ShipmentAssigned   of containerId : string
```

Comment:
are you sure there does not need to be a reserve vs assign phase?
otherwise there can be >1 of these
suggest at least putting a comment about this

Reply:
Yeah in our case we don't need a reserve state because the point in time where we consider an assignment to be valid is when we have the container finalized.

So if `shipmentA` and `shipmentB` have `assigned to containerA` events in their stream, the _assignment_ will only be valid when `containerA` has a `finalized containerA` event in its stream.

I've added a comment. Thanks!